CHANGELOG.md

```diff
@@ -15,3 +15,8 @@ First release.
 - Templates: MacVLAN launch-xrd example, xr-compose template
 - Sample xr-compose topologies: 'simple-bgp', 'bgp-ospf-triangle, 'segment-routing'
 - Tests: host-check UT
+
+
+### v1.0.1 (2022-09-06)
+
+Emit a warning in `host-check` when 2M hugepages are used.
```

Comment:
Minor point, but I'd suggest formatting as bullet points.
```suggestion
- Emit a warning in `host-check` when 2M hugepages are used.
```

Reply:
Maybe also be explicit that it's because this isn't a supported deployment use case.